packages/agent-nodejs-testing/src/integrations/testable-agent.ts

```diff
@@ -35,15 +34,12 @@ export default class TestableAgent<
   async start(): Promise<void> {
     await this.agent.mountOnStandaloneServer(this.agentOptions.port ?? 0).start();
     if (!this.agentOptions.schemaPath) throw new Error('schemaPath is required');
-
-    this.init({
-      actionEndpoints: SchemaConverter.extractActionEndpoints(
-        JSON.parse(await fs.readFile(this.agentOptions.schemaPath, 'utf8')),
-      ),
-      httpRequester: createHttpRequester({
-        url: `http://localhost:${this.agent.standaloneServerPort}`,
-        authSecret: this.agentOptions.authSecret,
-      }),
+    this.actionEndpoints = SchemaConverter.extractActionEndpoints(
```

Comment:
Is the SchemaConverter exposed ? This is definitely something we could use in the MCP server 😉